relay/p2p/relayerMessageHandler_test.go

```diff
@@ -223,12 +224,12 @@ func TestRelayerMessageHandler_createMessage(t *testing.T) {
 	})
 }
 
-func createSignedMessageAndMarshaledBytes() (*SignedMessage, []byte) {
+func createSignedMessageAndMarshaledBytes() (*core.SignedMessage, []byte) {
```

Comment:
create a function that generates n signedMessage

Reply:
simplified